install.sh

```
@@ -3,12 +3,12 @@
 #Copies the template to correct location and creates the intermediate dirs if neccessary
 
 TEMPLATE_PATH=~/Library/Developer/Xcode/Templates/File\ Templates/Source
-# Create the tempalte source folder if needed
+# Remove the old template source folder if needed
+rm -r "${TEMPLATE_PATH}"
```

We shouldn't do this I think, as it will also remove templates you installed from other sources than this repo.
In general, if the path is not fully under the control of the project, it's pretty scary to just completely remove the folder.

In the old solution this was not a real problem, because we were only removing templates that we would actually replace anyway.

I actually prefer to have a warning that the removed folder doesn't exist, instead of this. Perhaps to avoid the warning we could just add a flag to rm to fail silently, or pipe the response to null or smth.

---

I agree! I did not realize that its a shared folder 😅

> Perhaps to avoid the warning we could just add a flag to rm to fail silently, or pipe the response to null or smth.

I will apply this suggestion, thank you for the review and suggestion. 🙇